src/ui/pages/Login/Login.tsx

```diff
@@ -96,6 +95,7 @@ const Login = () => {
                     </p> 
                 </div >
             </div>
+
```

Comment:
You need to be more careful with adding all files to your commit. these are unnecessary changes because of it.